secator/exporters/txt.py

```
@@ -1,22 +1,35 @@
 			txt_path = f'{self.report.output_folder}/report_{output_type}.txt'
 			with open(txt_path, 'w') as f:
-				f.write('\n'.join(items))
+				first = True
+				for item in items:
+					if isinstance(item, dict):
+						cls = self._type_map.get(item.get('_type'))
+						if cls:
+							try:
+								item = cls.load(item)
+							except (TypeError, Exception):
+								pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)